src/backend/utils/indexer.js

```diff
@@ -13,15 +13,21 @@ const type = 'post';
  * Indexes the text and id from a post
  * @param text from a post
  * @param postId same id used to store on redis the post object where the text is from
+ * @param title from a post
+ * @param updated from a post
+ * @param author from a post
  */
-const indexPost = async (text, postId) => {
+const indexPost = async (text, postId, title, updated, author) => {
```

Comment:
Let's just pass the entire `post` object in, and then you can pick off the properties you want in this function vs. having to get the order of the arguments correct above.